exporters/stdout/stdoutlog/example_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package stdoutlog_test
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutlog"
+	"go.opentelemetry.io/otel/log/global"
+	"go.opentelemetry.io/otel/sdk/log"
+)
+
+func Example() {
+	exp, err := stdoutlog.New()
+	if err != nil {
+		panic(err)
+	}
+
+	processor := log.NewBatchProcessor(exp)
```

Comment:
`stdoutlog` is meant for debugging. Wouldn't it make more sense to change this to

```suggestion
	processor := log.NewSimpleProcessor(exp)
```

?